src/igvfd/types/sample.py

```diff
@@ -107,6 +122,21 @@ class InVitroSystem(Biosample):
     item_type = 'in_vitro_system'
     schema = load_schema('igvfd:schemas/in_vitro_system.json')
 
+    @calculated_property(
+        schema={
+            'title': 'Summary',
+            'type': 'string',
+            'notSubmittable': True,
+        }
+    )
+    def summary(self, request, biosample_term, taxa, classification=None, time_post_factors_introduction=None, time_post_factors_introduction_units=None):
+        sample_term_object = request.embed(biosample_term, '@@object')
```

Comment:
i am wondering if you need calculated properties when you embed the underlying sample_term or not, I think there is a way to skip calculated properties making it more efficient

Reply:
that is true for all of the calculated properties and embedding in them

Reply:
skipped calculated properties with '@@object?skip_calculated=true'